scripts/lighthouse/index.test.js

```diff
@@ -0,0 +1,20 @@
+import config from '../../lighthouse';
+
+import runLighthouse from './runLighthouse';
+import launchLighthouseAndLogResults from './index';
+import logResults from './logResults';
+
+jest.mock('./runLighthouse', () => jest.fn().mockResolvedValue(1));
+jest.mock('./logResults', () => ({
+  logHighLevelScores: jest.fn().mockResolvedValue(2),
+  checkFailures: jest.fn().mockResolvedValue(3),
+}));
+
+describe('index', () => {
+  it('launchLighthouseAndLogResults', async () => {
+    await launchLighthouseAndLogResults();
+    expect(runLighthouse).toHaveBeenCalledWith(config);
+    expect(logResults.logHighLevelScores).toHaveBeenCalledWith(1);
+    expect(logResults.checkFailures).toHaveBeenCalledWith(2);
```

Comment:
From an outsider's perspective - why are these values `1` and `2`? Looks like they should be the same as the `mockResolvedValue` calls (which would be `2` and `3` respectively)

Reply:
Because L17 needs the value returned from L, and L18 needs L9
The tests for this file are looking at whether the three functions in the promise chain have been called.

Reply:
Ohhhh ok. That took a while to click!

Perhaps rename `it('launchLighthouseAndLogResults'` -> `it('launches lighthouse, then logs scores, then checks for failures'` - and move the mocks inside the `it` block? (would be a bit more self-documenting)

Reply:
Good point about the test name I have updated it. Because there is only one group of related tests in the describe referencing them (the tests are doing one thing) it's probably OK to leave the mocks where they are. It requires a refactor to move the mocks into the describe block, not immediately clear how that should happen.